aesara/_version.py

```diff
@@ -16,6 +16,7 @@
 import sys
 from typing import Dict
 
+
```

Comment:
This change isn't necessary.

Reply:
Black automatically applied it. Ill remove it